src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -2407,6 +2396,23 @@ public Response getContentAccessBody(@Verify(Consumer.class) String consumerUuid
             throw new BadRequestException(i18n.tr("Cannot retrieve content access certificate"));
         }
 
+        // The date provided by the client does not preserve milliseconds, so we need to round down
+        // the dates preserved on the server by removing milliseconds for a proper date comparison
+        Date scaCertificateLastUpdated = cac.getUpdated() != null ?
+            Util.roundDownToSeconds(cac.getUpdated()) :
```

Comment:
Does this work as a stand-in for the format used by HTTP headers (and the formatter removed above)? IIRC, the formats are slightly different from typical time-related ISO/RFC formats.